mllib/src/main/scala/org/apache/spark/mllib/api/python/GaussianMixtureModelWrapper.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.mllib.api.python
 
-import scala.collection.JavaConverters
+import scala.jdk.javaapi.CollectionConverters.asJava
```

Comment:
```
import scala.jdk.CollectionConverters._
...
SerDe.dumps(modelGaussians.toSeq.asJava)
```?

Reply:
nit: looks like some places are importing `scala.jdk.javaapi.CollectionConverters.asJava`. Can we standardize to use `scala.jdk.CollectionConverters`?